plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -281,6 +294,1145 @@ public void testWriteDifferentSizes()
         computeActual(format("DROP TABLE %s", testTable));
     }
 
+    @Test
+    public void testPartitionProjectionOnVarCharColumnWithSingleEnumProjectionAndColumnNameWithWhitespace()
```

Comment:
nit: s/VarChar/Varchar (here and elsewhere)

Reply:
and make it shorter maybe: `testEnumPartitionProjectionOnVarcharColumnWithWhitespace`

Reply:
Applied this pattern to all test methods